src/main/handlers/openProjectHandler.ts

```diff
@@ -46,13 +46,18 @@ const openProjectFromFile: (filePath: string) => Promise<Project> = async (
 };
 
 const handleOpenProject: (
+  filePath: string | null,
   mainWindow: BrowserWindow | null
-) => Promise<{ project: Project; filePath: string }> = async (mainWindow) => {
-  const filePath = await getOpenFilePath(mainWindow);
+) => Promise<{ project: Project; filePath: string }> = async (
+  filePath,
+  mainWindow
+) => {
+  const openFilePath =
+    filePath === null ? await getOpenFilePath(mainWindow) : filePath;
```

Comment:
IIRC there's a coalescing operator that you could use here that'll shorten this.

```suggestion
    filePath ?? await getOpenFilePath(mainWindow);
```